pages/new.tsx

```diff
@@ -27,7 +29,7 @@ export default function NewFilePage() {
   const { userData, firebaseUser } = useNullableUserContext();
   const router = useRouter();
   const [lang, setLang] = useState<Language>('cpp');
-  const [fileName, setFileName] = useState('');
+  const [fileName, setFileName] = useState(generateRandomFileName());
   const [defaultPerimssion, setDefaultPermission] = useState<
```

Comment:
I think it may be better to pass generateRandomFilename as an initializer function (ie. without the parentheses) so it doesn't run every time the component is rerendered.

Reply:
Should be fixed, I added an `() => `